packages/ui-kit/src/components/markdown.tsx

```diff
@@ -386,7 +386,7 @@ const withCopyToClipboard =
           onClose={handleTooltipClose}
         >
           <CopyArea onClick={handleCopyToClipboardClick}>
-            <RibbonSvgIcon />
+            <ClipboardIcon size="big" color="neutral60" />
```

Comment:
"neutral60" is the most suited color provided by the ui-kit for their icons...also very close to the color code you provided @zbalek. And I assume all of their color codes for icons are following the WCAG. But maybe you can double check if the color is fine.
https://uikit.commercetools.com/?path=/story/basics-tokens--all-tokens (color-neutral-60)